src/js/actions.js

```diff
@@ -170,14 +160,8 @@ export function submitForm(formConfig, form) {
         // overly cautious
         const error = errorReceived instanceof Error ? errorReceived : new Error(errorReceived);
         const errorMessage = String(error.message);
-        let errorType = 'clientError';
-        if (errorMessage.startsWith('vets_throttled_error')) {
-          errorType = 'throttledError';
-        } else if (errorMessage.startsWith('vets_server_error')) {
-          errorType = 'serverError';
-        }
-        captureError(error, errorType);
-        dispatch(setSubmission('status', errorType, error.extra));
+        telemetry('error', error);
```

Comment:
These are obviously errors specific to vets.gov, so I pulled this out. If vets.gov needs site-specific messages they could be sent by the hook function. (Essentially, move the deleted code to the hook function I think,)